subprojects/packagefiles/rdkafka/meson.build

```diff
@@ -201,7 +208,7 @@ endif
 rdkafka = library('rdkafka', srcc, install: true, dependencies: rdkafka_deps)
 rdkafka_dep = declare_dependency(
   link_with: rdkafka,
-  include_directories: include_directories('include-workaround-meson'),
+  include_directories: '.',
```

Comment:
hrm unfortunate.

Reply:
Is `include_directories: '.'` necessary for anything?

Reply:
No, because all it does is allow you to `#include <src/........>` and compile against it using this declare_dependency.

Which is broken because it doesn't provide compatibility with `#include <librdkafka/.......>`